tests/plugin_filter/test_plugin_filter.py

```diff
@@ -113,30 +113,26 @@ def test_plugin_filter_name(tmp_db, client, template_tab, plugins, name: str):
         create_plugin(tmp_db, name=name)
 
     # test single name filter
-    tab_plugins = update_plugin_filter(tmp_db, client, template_tab, {"name": name})
+    tab_plugin_ids = update_plugin_filter(tmp_db, client, template_tab, {"name": name})
     filtered_plugin_ids = {
         p_id for p_id, in tmp_db.session.query(RAMP.id).filter(RAMP.name == name).all()
     }
-    assert len(tab_plugins) > 0
-    assert len(tab_plugins) == len(filtered_plugin_ids)
-    for plugin in tab_plugins:
-        assert plugin.id in filtered_plugin_ids
+    assert len(tab_plugin_ids) > 0
+    assert tab_plugin_ids == filtered_plugin_ids
```

Comment:
one last thing: add a helpful message to the asserts, e.g., "filtering by a single name failed"
The messages can either say what has failed (but do not describe just the assert statement...) or where the failure can be expected (only if you have a rough idea from the detected error already)